### ISSUE TITLE
[BUG] Report returning ruleId instead of message (en-US).

### ISSUE THREAD

**Issue by ricardoheil** [IBMa/equal-access#967]
Hi there,

I'm trying to call getCompliance method from accessibility-checker package using node.js, passing a the path of a local file.
`const aChecker = require('accessibility-checker');`
`aChecker.getCompliance('file://c:\test.html', 'id123').then((res) => console.log(res.report));`

Turns out the report is bringing the ruleId as messages, instead of the message itself. Example:
`{`
`	"ruleId": "WCAG20_Body_FirstASkips_Native_Host_Sematics",`
`	...`
`	"message": "WCAG20_Body_FirstASkips_Native_Host_Sematics"`
`}`

Debugging a little bit, I've identified that behind the scenes a puppeteer instance is created to load the specified file, and ace.js is loaded on it. Finally, when checker is created inside puppeteer, the method below returns empty, which causes the result to bring the ruleId as message.
https://github.com/IBMa/equal-access/blob/9c2ed8d909086fe65f2a76939e469e169c3bdfea/accessibility-checker-engine/src/v4/checker/Checker.ts#L30

I don't see a way to set the language (not even en-US) via accessible resources on my application.

**Comment by shunguoy** [comment]
@ricardoheil   Could you please run your code with the latest version of the accessibility-checker package?  Multiple issues were fixed and the dependencies were updated since.  Hopefully, the error you encountered was fixed (I am not able to duplicate the error with the current package).  thanks.

**Comment by shunguoy** [comment]
close as it cannot be duplicated with the current code
